eq3bt/eq3btsmart.py

```diff
@@ -11,6 +11,7 @@
 import struct
 import codecs
 from datetime import datetime, timedelta, time
+from construct import Byte
```

Comment:
This is an unused import, right?

Reply:
No, actually this is used in ``Thermostat.set_away``. ``time`` on the line above was however not used. That was shadowed by the star import from ``structures`` which has similar imports from ``datetime``.
I replaced the star import with classes from ``structures`` that are actually used, and removed the unused import from ``datetime``.